media-libs/mlt/mlt-7.14.0.ebuild

```diff
@@ -84,6 +84,7 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-6.10.0-swig-underlinking.patch
 	"${FILESDIR}"/${PN}-6.22.1-no_lua_bdepend.patch
 	"${FILESDIR}"/${PN}-7.0.1-cmake-symlink.patch
+	"${FILESDIR}"/${PN}-7.14.0-int-conversion.patch
```

Comment:
You'll need to git mv for this as it affects code generation.

Reply:
Done.